.github/styles/Vocab/default/accept.txt

```diff
@@ -52,6 +56,8 @@ Evmos
 Fantom
 Fuji
 Fuse
+gasless
+Gasless
```

Comment:
Maybe it is a good idea to make all the entries case-insensitive by adding the prefix `(?i)`: https://vale.sh/docs/topics/vocab/#case-sensitivity

In this case it would be `(?i)gasless` but we could add it to every word in the list.